bin/cdxgen.js

```diff
@@ -565,6 +570,32 @@ const checkPermissions = (filePath) => {
     options.usagesSlicesFile = `${options.projectName}-usages.json`;
   }
   prepareEnv(filePath, options);
+
+  if (options.inspectPurl) {
+    try {
+      const purlBomGenerator = await import("../lib/helpers/purlbom.js");
+      
+      let purlObj;
+      try {
+        purlObj = PackageURL.fromString(options.inspectPurl);
+      } catch (ex) {
+        console.error("Invalid PURL provided:", options.inspectPurl);
+        process.exit(1);
+      }
+
+      const pomPath = await purlBomGenerator.generatePurlMetadata(purlObj);
+      if (!pomPath) {
+        console.error("Failed to process the PURL:", options.inspectPurl);
+        process.exit(1);
+      }
+  
+      filePath = dirname(pomPath);
```

Comment:
We may have to do some post-processing to set a property or an evidence to the original purl that led to an sbom?